projects/project_tests.py

```diff
@@ -115,6 +115,19 @@ def test__projects__admin__ProjectResource__1(browser, project_dict):
     assert str(s2) in str(browser.contents)
 
 
+def test__projects__admin__ProjectResource__2(browser, project):
+    """Export begin and end date in correct timezone."""
```

Comment:
How do you check the correct timezone here? It currently only looks like it checks that there is a time at all.

Reply:
If the export wouldn't be correct, the projects start time for example would be set to 9:00 during the creation of the project object and when persisting set to 7:00 UTC and exported as 7:00 UTC. So only if the datetime gets exported in the correct time zone the start time of the object the export are the same.